public/js/icinga/behavior/flyover.js

```diff
@@ -0,0 +1,48 @@
+/* Icinga Web 2 | (c) 2017 Icinga Development Team | GPLv2+ */
+
+/**
+ * Icinga.Behavior.Flyover
+ *
+ * A toggleable flyover
+ */
+(function(Icinga, $) {
+
+    'use strict';
+
+    var expandedFlyovers = [];
+
+    function Flyover(icinga) {
+        Icinga.EventListener.call(this, icinga);
+
+        this.on('rendered', this.onRendered, this);
+    }
+
+    Flyover.prototype = new Icinga.EventListener();
+
+    Flyover.prototype.onRendered = function(event) {
+        $(event.target).find('.flyover').each(function() {
+            var $this = $(this);
+
+            if (expandedFlyovers.indexOf($this.attr('id')) > -1) {
+                $this.toggleClass('flyover-expanded');
+            }
+        });
+    };
+
+    Icinga.Behaviors = Icinga.Behaviors || {};
+
+    Icinga.Behaviors.Flyover = Flyover;
+
+    $(document).on('click', '.flyover .flyover-toggle', function(event) {
```

Comment:
Register this event handler in the behavior's constructor with `this.on()` instead of using `$(document)` directly.